src/ext/ddtrace.c

```diff
@@ -1546,3 +1551,7 @@ ZEND_GET_MODULE(ddtrace)
 ZEND_TSRMLS_CACHE_DEFINE();
 #endif
 #endif
+
+// the following operations are performed in order to put the tracer in a state when a new trace can be started:
+//   - set a new trace (group) id
+void _ddtrace_prepare_for_new_trace(TSRMLS_D) { DDTRACE_G(traces_group_id) = ddtrace_coms_next_group_id(); }
```

Comment:
To match my other suggestion; hopefully this +1 char doesn't push it over the line limit :)

```suggestion
static void dd_prepare_for_new_trace(TSRMLS_D) { DDTRACE_G(traces_group_id) = ddtrace_coms_next_group_id(); }
```